<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Science</class>
 <widget class="QWidget" name="Science">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="btnInverse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>1/x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>e</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnClearAll">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnExp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>exp</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnDel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/E:/image/cancel.png</normaloff>:/E:/image/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="btnMod">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>mod</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnAbs">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>|x|</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnPI">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>π</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn2nd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>2nd</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="5">
      <widget class="QLineEdit" name="display">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 20pt &quot;Bahnschrift SemiBold&quot;;
QLineEdit{
 color:#000;
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnSquare">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>x²</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnSqrt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>2√x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnXy">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>x^y</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="btn10x">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>10^x</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="btnLog">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="btnLn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="btnNum0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="btnZhengFu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="btnPeriod">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="btnNum3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btnNum2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btnNum1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="btnNum4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="btnNum5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="btnNum6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnNum8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnNum7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnNum9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnLeft">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnRight">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnN1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>n!</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="btnEqual">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(0, 95, 186);
}

QPushButton::hover{
	background-color: rgb(49, 125, 197);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(49, 125, 197);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="btnPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="btnMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="btnMultiple">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="btnDivide">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
